media/redesign/stylus/components/wiki/revision-list.styl

```diff
@@ -97,3 +97,7 @@ html[dir=rtl] .revision-list-controls:before {
     width: 460px;
 }
 
+/* make the Compare selected revisions button align to layout */
+.revision-list-controls .fixed {
+    left: 24px;
```

Comment:
I had to jump through so many hoops to get the internationalization stuff set up on my testing machine >_<  I'm happy to test the RTL :)